modelbaker/generator/generator.py

```diff
@@ -180,12 +182,17 @@ def layers(self, filter_layer_list: list = []) -> list[Layer]:
             if not relevant_topics and base_topic and base_topic.count(".") > 0:
                 relevant_topics.append(base_topic)
 
-            # Get table name in this order: translation if exists,
-            # alias (dispName) if exists, or ili_name.
-            alias = record.get("table_tr", None)
+            # If raw_naming is True, the layername should be the tablename
+            # Otherwise get table name in this order:
+            # - translation if exists,
+            # - alias (dispName) if exists
+            # - otherwise get_iliname
+            # And if it's a duplicate then check to consider geometry column
+            alias = None
+            if self.raw_naming:
+                alias = record.get("tablename")
             if not alias:
-                alias = record.get("table_alias", None)
```

Comment:
I am missing this option in the new code for layers.

I see you consider it for fields, as expected.

Reply:
oh yes. thanks a lot!